### PR TITLE
Update `Java: ArrayList` Concept With `.set()` Method Entry

### PR DESCRIPTION
**Pull request by DavidDeme**
### Type of Change

- Adding a new entry for the `.set()` method under the Java: ArrayList concept

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/java/concepts/array-list/terms/set/set.md
Comment (SSwiniarski) [inline content/java/concepts/array-list/terms/set/set.md:15]:
```suggestion
The **`.set()`** method is used for replacing the element present in a specified position with another element in instances of the `ArrayList` class. After execution, returns the replaced element's value.
```

File: content/java/concepts/array-list/terms/set/set.md
Comment (SSwiniarski) [inline content/java/concepts/array-list/terms/set/set.md:19]:
```suggestion
An element can be replaced in an `ArrayList` instance with another element, by passing new value to the `.set()` method with a specified index:
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@DavidDeme I did a first review and left a couple of comments.

File: content/java/concepts/array-list/terms/set/set.md
Comment (DavidDeme) [reply content/java/concepts/array-list/terms/set/set.md]: @SSwiniarski Hello, I have added all the suggestions

File: content/java/concepts/array-list/terms/set/set.md
Comment (SSwiniarski) [inline content/java/concepts/array-list/terms/set/set.md:19]:
```suggestion
An element can be replaced in an `ArrayList` instance with another element by passing new value to the `.set()` method with a specified index:
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@DavidDeme I had one more comment.

File: content/java/concepts/array-list/terms/set/set.md
Comment (DavidDeme) [reply content/java/concepts/array-list/terms/set/set.md]: @SSwiniarski  Updated...

**Review by SSwiniarski — APPROVED** [review]
@DavidDeme looks good for a second review.

File: content/java/concepts/array-list/terms/set/set.md
Comment (Dusch4593) [inline content/java/concepts/array-list/terms/set/set.md:20]:
```suggestion

```

File: content/java/concepts/array-list/terms/set/set.md
Comment (Dusch4593) [inline content/java/concepts/array-list/terms/set/set.md:15]:
```suggestion
The **`.set()`** method replaces an element at a specified position with another element in an `ArrayList` instance. After execution, the replaced element is returned.
```

File: content/java/concepts/array-list/terms/set/set.md
Comment (Dusch4593) [inline content/java/concepts/array-list/terms/set/set.md:22]:
```suggestion
arrayListInstance.set(index, newElement);
```

File: content/java/concepts/array-list/terms/set/set.md
Comment (Dusch4593) [inline content/java/concepts/array-list/terms/set/set.md:24]:
```suggestion

An element at a specified `index` can be replaced in an `arrayListInstance` with the `newElement`.

> **Note:** The `newElement` must be of the same [data type](https://www.codecademy.com/resources/docs/java/data-types) as the rest of the elements in `arrayListInstance`. Otherwise, an error will occur.

```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@DavidDeme Left some comments/suggestions for you. 😄

**Comment by DavidDeme** [comment]
> @DavidDeme Left some comments/suggestions for you. 😄

@Dusch4593 Cool suggestions, updated, please check 😄

**Review by Dusch4593 — APPROVED** [review]
@DavidDeme Looks good to merge! Thanks for pushing those changes. 😄

Before merging, I made some super small edits in ee43c6487b65e435a31f1de413b16d34278e0628 and c6db9becf56483002afb2c708d5ede330b467d46.

**Comment by Dusch4593** [comment]
Congrats and thank you for contributing to Codecademy Docs, @DavidDeme! 🎊 🥳 🎉 

The new entry should now be live: https://www.codecademy.com/resources/docs/java/array-list/set

I looks like your Codecademy is hidden on the entry page. To make it visible, you'll need to do the following:

1. Ensure you are using the same email address for both your Codecademy and GitHub accounts.
2. Add your GitHub username to your Codecademy profile.
3. Log out of Codecademy.
4. Log back in through your GitHub account.

Screen Shot 2023-01-19 at 9 32 52 AM

Looking forward to your next Docs contribution! 😄

**Comment by DavidDeme** [comment]
@Dusch4593 Thank you for information and cooperation, will try to contribute more, love the platform.. 😄

P.S I logged in with my Github account but nothing has chanced, hopefully it takes some time

**Comment by Dusch4593** [comment]
> P.S I logged in with my Github account but nothing has chanced, hopefully it takes some time

I'll check the entry again in an hour or so to see if the update was applied. 😃

**Comment by Dusch4593** [comment]
> P.S I logged in with my Github account but nothing has chanced, hopefully it takes some time

Just to clarify, you are using the same email address for both your Codecademy account and your GitHub account, right?

**Comment by Dusch4593** [comment]
@DavidDeme never mind, there are you are! 😄 

Screen Shot 2023-01-19 at 10 57 55 AM

**Pull request merged**
